documentation/rails.md

```diff
@@ -0,0 +1,124 @@
+Rails configuration
+===================
+
+To use this file with Rails you have two choices:
+
+#### 1. Adding a new `webpack_asset_url` helper
+
+You'll need a helper to replace asset pipeline. Here is an example how you an do this:
+
+```Ruby
+module WebpackHelper
+  def webpack_asset_url(asset)
+    "/assets/#{manifest.fetch(asset)}"
+  end
+
+  def manifest
+    @manifest ||= JSON.parse(File.read('manifest.json'))
+  rescue
+    fail 'Please run webpack'
+  end
+end
+```
+
+This file would then be saved in `app/helpers/webpack_helper.rb` for example.
+
+Now you can use it like this:
+
+```HTML
+<img src="#{webpack_asset_url('logo.svg')}" alt="logo" />
+```
+
+to require your assets. [Here](https://github.com/infinum/webpack-rails-manifest-plugin/blob/master/example/webpack.config.js#L12) is an example on how to add a digest to a file.
+
+**Take note that you can't use Rails `image_tag`, `stylesheet_link_tag`, `javascript_include_tag` helper.**
```

Comment:
Take note -> Please note
**the** Rails [... helpers ...] helper**s**.

Also, maybe elaborate on the **can't** use? Do you mean that these helpers do not integrate with this plugin and will not function as expected?